pandas/core/frame.py

```diff
@@ -2187,7 +2187,7 @@ def to_parquet(
             compression=compression,
             index=index,
             partition_cols=partition_cols,
-            **kwargs
+            **kwargs,
```

Comment:
is this a `black` version thing?

Reply:
Weirdly using 19.3b0 locally seems to add this for me - I've reverted

Reply:
Black code check in CI fail without this - will add back in

Reply:
Personally, I find this quite odd that you have to add a comma at the end.

Reply:
Related to this - https://github.com/pandas-dev/pandas/pull/29607